tools/cellprofiler/gray_to_color.xml

```diff
@@ -3,56 +3,55 @@
+            <param name="rgb_blue" label="Enter the name of the image to be colored blue" value="Leave this black" type="text"/>
+            <param name="rgb_blue_weight" label="Relative weight for the blue image" optional="true" type="float" min="0.0" max="1.0" value="1.0" />
 
-            <param name="name_output_image" label="Name the output image" value="ColorImage" type="text"/>      
+            <param name="name_output_image" label="Enter the name of the resulting image" value="ColorImage" type="text"/>
         </xml>
         <xml name="cmyk">
-            <param name="cmyk_cyan" label="Select the image to be colored cyan" value="Leave this black" type="text"/>
-            <param name="cmyk_cyan_weight" label="Relative weight for the cyan image" optional="true" type="float" value="0.5"/>
+            <param name="cmyk_cyan" label="Enter the name of the image to be colored cyan" value="Leave this black" type="text"/>
+            <param name="cmyk_cyan_weight" label="Relative weight for the cyan image" optional="true" type="float" min="0.0" max="1.0" value="1.0" />
 
-            <param name="cmyk_magenta" label="Select the image to be colored magenta" value="Leave this black" type="text"/>
-            <param name="rgb_green_weight" label="Relative weight for the green image" optional="true" type="float" value="0.0"/>
+            <param name="cmyk_magenta" label="Enter the name of the image to be colored magenta" value="Leave this black" type="text"/>
+            <param name="rgb_green_weight" label="Relative weight for the green image" optional="true" type="float" min="0.0" max="1.0" value="1.0" />
 
-            <param name="cmyk_yellow" label="Select the image to be colored yellow" value="Leave this black" type="text"/>
-            <param name="cmyk_yellow_weight" label="Relative weight for the yellow image" optional="true" type="float" value="0.0"/>
+            <param name="cmyk_yellow" label="Enter the name of the image to be colored yellow" value="Leave this black" type="text"/>
+            <param name="cmyk_yellow_weight" label="Relative weight for the yellow image" optional="true" type="float" min="0.0" max="1.0" value="1.0" />
 
-            <param name="cmyk_brightness" label="Select the image that determines brightness" value="Leave this black" type="text"/>
-            <param name="cmyk_brightness_weight" label="Relative weight for the brightness image" optional="true" type="float" value="0.5"/>
-
-            <param name="name_output_image" label="Name the output image" value="ColorImage" type="text">
+            <param name="cmyk_brightness" label="Enter the name of the image that determines brightness" value="Leave this black" type="text"/>
+            <param name="cmyk_brightness_weight" label="Relative weight for the brightness image" optional="true" type="float" min="0.0" max="1.0" value="1.0" />
+<!-- The param name_output_image is common to all 4 options, shouldn't be out of the conditional?-->

```

Comment:
this?

Reply:
changed.. will be updated in the next commit together with the multitiff option